### PR TITLE
add webnotification to ask user to join in room

### PR DESCRIPTION
**Pull request by juliemar**
How can I test this PR.

1) you have to be logged with 2 users in different rooms
2) Logged with "user B", execute right click on "user A".
2.a) should appear the Get menu.
2.b) click on "Get" Menu
Result: In "User A" Browser should appear a web notification and a confirmation prompt with this message "User B is calling you to join in Room X"

### CODE REVIEW

File: public/js/office.web.js [hunk 1]
```diff
@@ -163,16 +163,20 @@ $(() => {
       icon: user.imageUrl,
     };
 
+    const loggedUserId = matrixProfile.loadStoredProfile().id;
+    const loggedUserRoomId = getLastRoom(matrixProfile);
+
+    if (loggedUserRoomId == roomId && loggedUserId != user.id) {
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
    if (loggedUserRoomId === roomId && loggedUserId !== user.id) {
```

File: public/js/office.web.js [hunk 2]
```diff
@@ -223,11 +227,21 @@ $(() => {
   }
 
   function confirmRoomEnter(user,roomId, officeEvents){
-    var isConfirmed = confirm(user.name +" is calling you to join in "+ getRoomName(roomId));
+
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
```

File: public/js/office.web.js [hunk 3]
```diff
@@ -223,11 +227,21 @@ $(() => {
...
+    const options = {
+      icon: user.imageUrl,
+    };
+
+    text = user.name+" is calling you to join in "+getRoomName(roomId);
```
Comment (lccezinha) [inline public/js/office.web.js]:
```suggestion
    text = `#{user.name} is calling you to join in #{getRoomName(roomId)}`;
```

**Comment by lccezinha** [comment]
@juliemar Code seems ok, how can I test the feature? :)

**Comment by juliemar** [comment]
you have to be logged with 2 users in different rooms
Logged with "user B", execute right click on "user A".
2.a) should appear the Get menu.
2.b) click on "Get" Menu
Result: In "User A" Browser should appear a web notification and a confirmation prompt with this message "User B is calling you to join in Room X"

**Comment by lccezinha** [comment]
@juliemar 

I have to do a small fix in the interpolation string, now everything is working fine! :rocket:

**Pull request merged**